spec/features/gardens/destroy_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Delete garden functionality' do
+  describe 'a logged in user' do
+    before :each do
+      @user_with_gardens = User.new({id: 4,
+      attributes: {
+        email: 'planter@gmail.com' },
+      relationships: {
+        gardens: {
+          data: [ {id: '3', type: 'garden'}, {id: '4', type: 'garden'}] }}})
+
+      garden1 = File.read('spec/fixtures/public_garden.json')
+      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/gardens/3").to_return(status: 200, body: garden1)
+
+      garden2 = File.read('spec/fixtures/private_garden.json')
+      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/gardens/4").to_return(status: 200, body: garden2)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user_with_gardens)
+    end
+
+    xit 'can delete a garden from the dashboard' do
```

Comment:
Is this skipped on purpose?

Reply:
Yes this is the test I was referencing on my PR description :(